engine/access/rpc/backend/backend_scripts.go

```diff
@@ -99,15 +100,20 @@ func (b *backendScripts) executeScriptOnExecutionNode(
 	// encode to MD5 as low compute/memory lookup key
 	encodedScript := md5.Sum(script) //nolint:gosec
 
+	// record start time
+	execStartTime := time.Now()
```

Comment:
I think the start time should be taken within the loop so it only includes the time spent executing on the EN that returned a successful result.

Reply:
Fixed

Reply:
@peterargue Do we not want to take into account the time it takes us to find a successful execution node ?

Reply:
no, this metric is looking to measure how long it takes to execute the script from the ANs perspective. We have other metrics that capture processing time of the gRPC endpoint that we could use for total time.